Dockerfile

```diff
@@ -440,6 +440,15 @@ ENV PATH=$PATH:/opt/php/bin
 RUN mkdir -p /opt/php/bin \
     && curl -ksS https://getcomposer.org/installer | php -- --install-dir=/opt/php/bin --filename=composer --$COMPOSER_VERSION
 
+# nuget-inspector
```

Comment:
> Commit message: Please add "analyzer" as the scope, i.e. `feat(analyzer): ...`.

But actually, not that I see that `NugetInspector.kt` is not a new analyzer on its own, the topic should probably also be "refactor" instead of "feat", like

    `refactor(analyzer): Use nuget-inspector to analyzse DotNet projects`

or so.

Reply:
Done!